src/index.html

```diff
@@ -2,15 +2,50 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="#"
+    method="post"
+    data-qa="big"
+    >
+
+      <label class="search">
+
+        <input
+        class="search__bar search__big"
+        type="text"
+        name="search__big"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        >
+
+      </label>
```

Comment:
- don't add empty lines between parent and child
- make 2 spaces indentation for attributes if you move them to a new line
```suggestion
      <label class="search">
        <input
          class="search__bar search__big"
          type="text"
          name="search__big"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </label>
```